changelogs/current.yaml

```diff
@@ -125,6 +125,12 @@ removed_config_or_runtime:
     Removed ``envoy.reloadable_features.normalize_host_for_preresolve_dfp_dns`` runtime flag and legacy code paths.
 
 new_features:
+- area: router check tool
```

Comment:
Taking a step back, can you explain how this is different than running Envoy in [validation mode](https://www.envoyproxy.io/docs/envoy/v1.31.0/start/quick-start/run-envoy.html#validating-your-envoy-configuration)?
That should take into account the runtime-flags, and validate the routes, no?

Reply:
Unless I'm missing something I think validation mode serves a different purpose than [router check tool](https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/tools/router_check). It is mostly used to assert through test files that envoy will route to the correct cluster, among other things. To give more context I would like to replace our [current validation tool](https://github.com/getyourguide/istio-config-validator/tree/main) with router check tool instead.